tests/lib/cli-engine.js

```diff
@@ -1457,12 +1535,13 @@ describe("CLIEngine", function() {
 
                 engine.executeOnFiles([file]);
 
-                assert.isTrue(fs.existsSync(path.resolve(".eslintcache")), "the cache for eslint was created");
+                assert.isTrue(fs.existsSync(cacheFile), "the cache for eslint was created");
 
                 engine = new CLIEngine({
                     useEslintrc: false,
                     // specifying cache true the cache will be created
                     cache: false,
+                    cacheFile: cacheFile,
```

Comment:
@nzakas It was a bug we did not detect before, because the default `.eslintignore` file was the right one until we switched the `cwd` for the tests. We could also not see the test fail after my first changes, because during the tests the default options got overwritten - @gyandeeps [pointed that out in an earlier comment](https://github.com/eslint/eslint/pull/3948#discussion_r48548239). IMO the change belongs here, but I can move that line to another PR if you prefer.